src/utils.tsx

```diff
@@ -0,0 +1,8 @@
+export function stringToEnum(str: string, enumType: any): any {
```

Comment:
I created this utility which I use in a number of places to turn `enumerated` MobX state into `enum`s that are typed correctly. I couldn't find anyone else doing this, which makes me suspicious of the approach. What do you think?

Reply:
I think it's fine. On a separate topic, this file doesn't contain JSX and probably shouldn't, so it can have a `.ts` extension.